identity/app/views/consentJourney.scala.html

```diff
@@ -126,19 +121,20 @@
 @marketingConsentStep = @{
     ConsentStep(
         name = "consent",
-        title = "What would you like to hear about?",
-        help = List(ConsentStepHelpText("Just take a couple of minutes to tell us what you’re interested in, so we can send you emails that we think you’ll find useful.")),
+        title = "Would you like to hear about any of these Guardian products?                                                    ",
```

Comment:
personally I'm not a fan of the word "product" here but I know that's not your call! Just point it out.

Reply:
![05onfire1_xp-facebookjumbo](https://user-images.githubusercontent.com/11539094/35274192-1a397c5e-0033-11e8-8d96-f0985fdea3d0.jpg)

FWIW I want @stephanfowler to have a look before shipping this, there's a couple things up for discussion from Monday's catchup!